apis/logging/v1/output_types.go

```diff
@@ -225,3 +228,32 @@ type Loki struct {
 	// +optional
 	LabelKeys []string `json:"labelKeys,omitempty"`
 }
+
+// GoogleCloudLogging provides configuration for sending logs to Google Cloud Logging
+type GoogleCloudLogging struct {
+	// Only one of BillingAccountID, OrganizationID, FolderID, or ProjectID can be used to send logs.
+	// If more than one are configured, the priority is as follows
+	// 1. BillingAccountID
+	// 2. OrganizationID
+	// 3. FolderID
+	// 4. ProjectID
+	//
+	// Reference: https://cloud.google.com/billing/docs/concepts
+
+	// +optional
+	BillingAccountID string `json:"billing_account_id,omitempty"`
```

Comment:
Why are these fields in the API instead of auth keys as part of the secret?  Are these not secrets that should be hidden from the API?

Reply:
no, what goes in the secret is service account token. It contains `private_key` , and `private_key_id`